app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
I think the above notes are enough to get you started- part of the challenge is that you're trying to handle two kinds of error in one place, and as a side effect of that, you're only checking for known errors.  (all the actions in the catch block only fire when a statement is true; there's no fallback)

I'm happy to look at concrete ideas, but anything more would be overspecifying- just note that currently it doesn't cover known kinds of error we saw yesterday.

There are some well-defined places to deal with errors in ember. The below may, or may not, be useful:
https://guides.emberjs.com/v2.5.0/routing/loading-and-error-substates/#toc_the-code-error-code-event